robot_servicio/scripts/move.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+import math
+import rospy
+import actionlib
+from move_base_msgs.msg import MoveBaseAction, MoveBaseGoal
+from geometry_msgs.msg import Pose, Quaternion
+from tf.transformations import quaternion_from_euler
+from std_msgs.msg import String
+import send_goals
+
+
+class receive_goals():
+    def __init__(self):
+        self.new_pose = Pose()
+        self.received_goals = []
+        rospy.Subscriber("/create1/move_base_goals", String, self.register_goal)
```

Comment:
Este approach no va a funcionar con múltiples robots, por lo cual tienen que eliminar `create1` y agregar el namespace al launch file.

```suggestion
        rospy.Subscriber("move_base_goals", String, self.register_goal)
```